src/color.rs

```diff
@@ -367,6 +371,8 @@ define_colors! {
     pub struct Rgba<T: Primitive Enlargeable>([T; 4, 1]) = "RGBA";
     /// Grayscale colors + alpha channel
     pub struct LumaA<T: Primitive>([T; 2, 1]) = "YA";
+    /// CMYK
+    pub struct Cmyk<T: Primitive>([T; 4, 1]) = "CMYK";
```

Comment:
This would all be much more straightforward if we don't implement Cmyk as a type. Is it helpful for other things to have it or is it more of an annoyance?

Reply:
I'd rather not add this struct